src/core/init/multithread/worker/track_choice_setter.ts

```diff
@@ -122,6 +122,7 @@ export default class TrackChoiceSetter {
         // TODO
         relativeResumingPosition: undefined,
         representations: ref.representations,
+        relativeResumingPosition: choice.relativeResumingPosition,
```

Comment:
This is the line that breaks linting and tests.
You need to remove the other property with the same name I put as a placeholder here

Reply:
Yes I just see it, strangely I didn't had a git conflict on this line when rebasing 🤔